src/seedsigner/views/seed_views.py

```diff
@@ -323,19 +323,51 @@ def __init__(self):
 
 
     def run(self):
-        ret = self.run_screen(seed_screens.SeedAddPassphraseScreen, passphrase=self.seed.passphrase)
+        ret_passphrase, ret_btn = self.run_screen(seed_screens.SeedAddPassphraseScreen, passphrase=self.seed.passphrase)
 
-        if ret == RET_CODE__BACK_BUTTON:
-            return Destination(BackStackView)
-        
         # The new passphrase will be the return value; it might be empty.
-        self.seed.set_passphrase(ret)
-        if len(self.seed.passphrase) > 0:
+        self.seed.set_passphrase(ret_passphrase)
+        if ret_btn == RET_CODE__BACK_BUTTON:
+            if len(self.seed.passphrase) > 0:
+                return Destination(SeedAddPassphraseExitDialogView)
+            else:
+                return Destination(BackStackView)
+        elif len(self.seed.passphrase) > 0:
             return Destination(SeedReviewPassphraseView)
         else:
             return Destination(SeedFinalizeView)
 
 
+class SeedAddPassphraseExitDialogView(View):
+    EXIT = ("Exit", None, None, "red")
+    CONTINUE = "Continue editing"
+
+    def __init__(self):
+        super().__init__()
+        self.seed = self.controller.storage.get_pending_seed()
+
+
+    def run(self):
+        passphrase = self.seed.passphrase
+
+        button_data = [self.EXIT, self.CONTINUE]
+        
+        selected_menu_num = self.run_screen(
+            WarningScreen,
+            title="Exit",
```

Comment:
suggestion: "Abandon Passphrase?" as a title.

Reply:
Thanks @jdlcdl. 

I agree with this suggestion for both title and text, and leaving button texts as proposed.

Let's see if other contributors comment on this, if not then I'll push the changes you proposed (after checking that new title fits).

Reply:
I tested your suggestion. It fits and looks quite better.

![SeedAddPassphraseExitDialogView](https://github.com/SeedSigner/seedsigner/assets/50918598/3ef7ec55-e6a8-4bad-b54d-b3a1e50c07a9)

I also had the idea of switching the button order. This way, if you accidentally press the button, you will continue editing instead of exiting, which aligns with the spirit of this issue.

Reply:
Since still waiting for feedback from others, please feel free to adjust to the best pr that you can (text changes and button preferences as you like) w/ another commit and push.

I like the text just above, and the thought of an deliberate exit by "down" and "press" to avoid accidentally abandoning the long passphrase.

Reply:
In the psychology of the user what happens we the don't know what is BIP-39? Do we really need to add BIP-39. 
Are we building for ourself or for the next billion users? my two sats.

Reply:
Thank you for your comment, @dulcedu. I believe the use of BIP numbers throughout the UX is something that should be considered globally. In this particular case, I think it's good practice to keep "BIP-39 Passphrase" to maintain consistency with the overall UX design.

I think this point could unleash a broader debate to balance between design accessibility to a wide audience and adhering to best practices.

Reply:
I appreciate the thoughts on BIP-39, and its 100% fair perspective on it being a technical detail. There are unfortunately several kinds of passwords/phrases that can be implemented with wallets / private keys, so I think it makes sense to include it for clarity somewhere in the UI.